lib/Benchmark.t

```diff
@@ -40,6 +40,73 @@ sub cmp_delta {
     return $max/$min <= (1+$delta);
 }
 
+sub splatter {
+    my ($message) = @_;
+    my $splatter = <<~'EOF_SPLATTER';
+    Please file a ticket to report this. Our bug tracker can be found at
+
+        https://github.com/Perl/perl5/issues
+
+    Make sure you include the full output of perl -V, also uname -a,
+    and the version details for the C compiler you are using are
+    very helpful.
+
+    Please also try compiling and running the C program that can
+    be found at
+
+        https://github.com/Perl/perl5/issues/20839#issuecomment-1439286875
+
+    and provide the results (or compile errors) as part of your
+    bug report.
+
+    EOF_SPLATTER
+
+    if ( $message =~ s/\.\.\.//) {
+        $splatter =~ s/Please/please/;
+    }
+    die $message, $splatter;
+}
+
+{
+    # Benchmark may end up "looping forever" if time() or times() are
+    # broken such that they do return changing values over time. The
+    # following crude test is intended to ensure that we can reply on
+    # them sufficiently we don't infinite loop.
```

Comment:
* "such that they do" -> "such that they do not"
* "we can reply" -> "we can rely"
* "sufficiently we don't infinite loop" - "so that we don't infinite loop"

Reply:
Will fix. Thanks!

Reply:
Done.